src/tree_annotation/database.cljs

```diff
@@ -173,7 +173,16 @@
 
 (defn del-node [coord]
   (when (last-leaf-or-inner-node? coord)
-    (do (del-ancestors coord)
+    (do ; delete all ancestors of the node (parent, grand-parent, ...)
```

Comment:
no `do` necessary with `when`.

Reply:
Ahhh, I can never remember when I have to write do and when not. :) I'm sure there are a lot of other places in the code where there are unnecessary dos..